packages/sdk/snet/sdk/__init__.py

```diff
@@ -56,6 +59,47 @@ def create_service_client(self, org_id, service_id, service_stub, group_name=Non
         return service_client
 
 
+    def create_dynamic_service_client(self, org_id, service_id, group_name=None, payment_channel_management_strategy=PaymentChannelManagementStrategy, options=None):
```

Comment:
Could you give an example of using your SDK with dynamic classes? As usual better directly in the tests...

Reply:
Done, there's both tests and the new PR in the examples repo (https://github.com/singnet/snet-code-examples/pull/7)